bucket/everything-cmdpal.json

```
@@ -0,0 +1,49 @@
+    "version":"0.2.0.0",
+    "description":"Adds Everything search to Command Palette.",
+    "homepage": "https://github.com/lin-ycv/EverythingCommandPalette",
+    "license":"epl-2.0",
+    "suggest":{
+        "Everything":[
+            "extras/everything",
+            "versions/everything-alpha"
+        ],
+        "PowerToys": "extras/powertoys"
+    },
+    "autoupdate":{
```

Biome (2.1.2)

[error] The key autoupdate was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)